core/src/test/java/kafka/admin/ConfigCommandTest.java

```diff
@@ -1013,7 +1013,7 @@ public void shouldAlterTopicConfig(boolean file) {
         addedConfigs.put("delete.retention.ms", "1000000");
         addedConfigs.put("min.insync.replicas", "2");
         if (file) {
-            File f = kafka.utils.TestUtils.tempPropertiesFile(JavaConverters.mapAsScalaMap(addedConfigs));
+            File f = kafka.utils.TestUtils.tempPropertiesFile(CollectionConverters.MapHasAsScala(addedConfigs).asScala());
```

Comment:
Could you please import `scala.jdk.javaapi.CollectionConverters` instead then rewrite this to `CollectionConverters.asScala(addedConfigs)`